storm-data-contracts/src/main/java/com/forter/contracts/cache/CacheDAO.java

```diff
@@ -0,0 +1,14 @@
+package com.forter.contracts.cache;
+
+import com.google.common.base.Optional;
+
+/**
+ * DAO accessing cache for external service results.
+ */
+public interface CacheDAO<TInput, TOutput> {
+
+    public Optional<TOutput> get(TInput key);
+
+    public void save(TOutput record, TInput input, long startTime);
```

Comment:
Fix the readme file to include a usage example. Point to the test where you show a simple implementation using hashmap.